tests/conftest.py

```diff
@@ -275,19 +277,29 @@ def handle_request(self, method, base_uri, timeout, headers=None, params=None, d
 
         if params:
```

Comment:
Could do like "data = params or data"

Reply:
tbh I wasted 3 days on this one, not making small changes to it lol